<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>applicantRegistration</class>
 <widget class="QWidget" name="applicantRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>766</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>596</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	
	background-color: rgb(20, 41, 63);
}

QLabel{
	
	font: 600 12pt &quot;Segoe UI&quot;;
}

QLineEdit{

	border: none;
	border-bottom: 1px solid white;
	
	font: 11pt &quot;Segoe UI&quot;;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">margin-top:20px;
font: 700 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>40</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Full Name:              </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameTxt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Email:                     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailTxt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nationality:            </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nationalityTxt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Phone Number     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneNumTxt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ageSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="genderCombo">
         <property name="styleSheet">
          <string notr="true">padding:7px;
font: 11pt &quot;Segoe UI&quot;;
</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string extracomment="Please Select From the List"/>
         </property>
         <property name="placeholderText">
          <string>Please Select From the List</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Date of Birth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DOBdateEdit">
         <property name="styleSheet">
          <string notr="true">padding:7px;
font: 11pt &quot;Segoe UI&quot;;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Education Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="educationCombo">
         <property name="styleSheet">
          <string notr="true">padding:7px;
font: 11pt &quot;Segoe UI&quot;;
</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string extracomment="Please Select From the List"/>
         </property>
         <property name="placeholderText">
          <string>Please Select From the List</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Password:              </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passTxt">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showHidePassBtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
margin-top:7px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../projectRescources.qrc">
           <normaloff>:/icons/seeWhiteIcon.png</normaloff>:/icons/seeWhiteIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Confirm Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirmPassTxt">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showHideConfirmPassBtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
margin-top:7px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../projectRescources.qrc">
           <normaloff>:/icons/seeWhiteIcon.png</normaloff>:/icons/seeWhiteIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorMsgLog">
     <property name="styleSheet">
      <string notr="true">color:Red;
margin-left:15px;
margin-right:15px;
margin-top:15px;</string>
     </property>
     <property name="text">
      <string>Error Msg Here....</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="signUpBtn">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(11, 100, 197);
font: 600 12pt &quot;Segoe UI&quot;;
padding:15px;
margin-left: 20px;
margin-right: 20px;</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="leftMargin">
      <number>145</number>
     </property>
     <property name="rightMargin">
      <number>200</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Already have an account?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goToLoginPageBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
font: 600 12pt &quot;Segoe UI&quot;;
text-decoration: underline;
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../projectRescources.qrc"/>
 </resources>
 <connections/>
</ui>
